hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -262,16 +257,17 @@ set_max_delay ${SPI_HIDO_PASS_MAX_DELAY} -from [get_ports SPI_HOST_D3] -to [get_
 #####################
 
 # this may need some refinement (and max delay / skew needs to be constrained)
-set_clock_groups -name group1 -async -group [get_clocks MAIN_CLK     ] \
-                                     -group [get_clocks USB_CLK      ] \
-                                     -group [get_clocks SPI_DEV_CLK  ] \
-                                     -group [get_clocks SPI_HOST_CLK ] \
-                                     -group [get_clocks IO_CLK       ] \
-                                     -group [get_clocks IO_DIV2_CLK  ] \
-                                     -group [get_clocks IO_DIV4_CLK  ] \
-                                     -group [get_clocks RNG_CLK      ] \
-                                     -group [get_clocks JTAG_TCK     ] \
-                                     -group [get_clocks AON_CLK      ]
+set_clock_groups -name group1 -async \
+    -group [get_clocks MAIN_CLK     ] \
+    -group [get_clocks USB_CLK      ] \
+    -group [get_clocks {SPI_DEV_CLK SPI_DEV_IN_CLK SPI_DEV_OUT_CLK} ] \
+    -group [get_clocks SPI_HOST_CLK ] \
```

Comment:
lgtm. just a note we may have to revisit spi_host_clk once `spi_host` actually lands.